src/taco-cli/test/templates.ts

```diff
@@ -13,14 +13,21 @@
 
 "use strict";
 
-var shouldModule = require("should"); // Note not import: We don't want to refer to shouldModule, but we need the require to occur since it modifies the prototype of Object.
+/* tslint:disable:no-var-requires */
+// var require needed for should module to work correctly
+// Note not import: We don't want to refer to shouldModule, but we need the require to occur since it modifies the prototype of Object.
+var shouldModule = require("should");
+/* tslint:enable:no-var-requires */
+
+/* tslint:disable:no-var-requires */
+// Special case to allow using color package with index signature for style rules
+var colors = require("colors/safe");
+/* tslint:enable:no-var-requires */
 
 import tacoUtils = require ("taco-utils");
 import Templates = require ("../cli/templates");
 import ms = require ("./utils/memoryStream");
 
-var colors = require("colors/safe");
```

Comment:
Actually I think this can already be an import; I just checked the colors.d.ts and they declare "colors/safe" as a module so it should pick it up fine.